synapse/federation/federation_server.py

```diff
@@ -1399,7 +1399,7 @@ async def on_edu(self, edu_type: str, origin: str, content: dict) -> None:
         # Check if we have a handler on this instance
         handler = self.edu_handlers.get(edu_type)
         if handler:
-            with start_active_span_from_edu(content, "handle_edu"):
+            with start_active_span_from_edu("handle_edu", edu_content=content):
```

Comment:
Do we get value out of this trace connection from the homeserver EDU sender to the homeserver that receives the EDU? Wondering about ripping it all out